lib/matplotlib/pyplot.py

```diff
@@ -1131,106 +1131,11 @@ def subplots(nrows=1, ncols=1, sharex=False, sharey=False, squeeze=True,
         # same as
         plt.subplots(2, 2, sharex=True, sharey=True)
     """
```

Comment:
Can we eliminate a lot of this docstring?  Either with ``@_autogen_docstring`` or ``%(Figure.subplots)s`` or something?